src/test/java/reposense/authorship/FileInfoExtractorTest.java

```diff
@@ -163,6 +166,18 @@ public void generateFileInfo_fileWithoutSpecialCharacters_correctFileInfoGenerat
         Assert.assertEquals(5, fileInfo.getLines().size());
     }
 
+    @Test
+    public void getListOfNonBinaryFiles_directoryWithBinaryAndNonBinaryFiles_success() {
+        GitCheckout.checkoutBranch(config.getRepoRoot(), BINARY_AND_NON_BINARY_FILES_BRANCH);
+        Set<Path> files = FileInfoExtractor.getListOfNonBinaryFiles(config);
+
+        Assert.assertEquals(1, files.size());
+        // Non binary files should be captured
+        Assert.assertTrue(files.contains(Paths.get("binaryFileTest/nonBinaryFile.txt")));
+        // Binary files should be ignored
+        Assert.assertFalse(files.contains(Paths.get("binaryFileTest/binaryFile.txt")));
```

Comment:
try to make the test a little more exhaustive, adding in images, pdf n stuffs.
Also a few more directories and sub folders will be good as well, we need to make sure the implementation is recursive as well.

For files test, we should at least aim for min 10 files.